scripts/local/starterkit/create-export

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/env bash
+
+# The idea is that you would clone the yalesites-starterkit repository to
+# a directory, and then run this script, passing the directory to that
+# repository.  Provided environment variables were set, it will:
+#   - Create the export remotely on the yalesites server
+#   - Retrieve the zip file and delete it from the server
+#   - Unzip it to the repository location
+#   - Create a commit with a timestamp
+#   - Delete the original zip file created
+# 
+# Environment variables needed:
+#   - PANTHEON_NAME: The terminus site name to connect to
+#   - SFTP_PORT: the port to use when SFTPing
+#   - SFTP_URI: the username@location to SFTP to
+# 
+# Usage:
+#   PANTHEON_NAME=ys-starterkit.dev SFTP_PORT=2222 \
+#   SFTP_URI="uritothelocation" ./scripts/local/starterkit/create-export \
+#   /code/yalesites-starterkit/starterkit
+
+# Since this script will attempt to pull the zip from the URL, the SFTP
+# settings must be set beforehand.
+verifyEnvironmentVariables() {
+    declare -a env_vars=("SFTP_URI" "SFTP_PORT" "PANTHEON_NAME")
+
+    for env_var in "${env_vars[@]}"
+        do
+            if [[ -z "${!env_var}" ]]; then
+                echo "Environment variable $env_var is not set. Please pre-set this varaible before running."
+                echo "Environment variables needed: ${env_vars[*]}"
+                exit 1
+            fi
+        done
+}
+
+export-content() {
+    local zipRemoteLocation
+    local zipFilename
+    local destinationFolder="${1:-$(mktemp -d)}"
+
+    zipRemoteLocation="$(retrieveZipLocation "$(runExport)")"
+    zipFilename="$(basename "$zipRemoteLocation")"
+    retrieveFileFromSftp "$zipRemoteLocation"
+    if [[ -f "$zipFilename" ]]; then
+        unzipToFolder "$zipFilename" "$destinationFolder"
+        createCommit "$destinationFolder"
+        removeZipFile "$zipFilename"
+    else
+        exit 1
+    fi
+
+    echo "Content exported to"
+    echo "$destinationFolder"
+}
+
+# Figures out the zip location on the remote server based on the stdout of the
+# terminus command.
+retrieveZipLocation() {
+    local zipFilename
+    local tmpPath="/tmp/export/zip"
+
+    # We don't have permission to create the scs-export directory under
+    # /code/web, so it errors out.  We redirect the error to stdouot so we can
+    # capture the file name that was attempted.  We then use the tmp directory
+    # location that it was when it was created, which still exists.
+    zipFilename=$(echo "$1" | head -5 | tail -1 | grep -o "temporary://[^']*" | xargs -I {} basename {})
+    echo "$tmpPath/$zipFilename"
+}
+
+runExport() {
+    terminus drush "$PANTHEON_NAME" -- content:export --all-content --assets --translate 2>&1
```

Comment:
Consider renaming these to follow other naming conventions for other scripts. For exmaple 'machine name'.